app/controllers/api/v1/books_controller.rb

```diff
@@ -1,30 +1,25 @@
 class Api::V1::BooksController < ApplicationController
+  before_action :authenticate_user!
   before_action :set_book, only: %i[ show update destroy ]
-
-  # GET /books
+  
   def index
-    @books = Book.all
-
+    @books = current_user.books
```

Comment:
@nshelokar-vf Why are initialising with instance variable here? Are we using this elsewhere outside the method scope?

Reply:
No sir, not using it elsewhere. Removed it now.